docs/natspec-format.rst

```diff
@@ -57,6 +57,8 @@ The following example shows a contract and a function using all available tags.
   public. You are welcome to use similar comments for your internal and
   private functions, but those will not be parsed.
 
+  This may change in the future.
```

Comment:
We may expose internal functions.

Reply:
Is this part of the `note`?

Reply:
Yes. Checked on a local build and it works too.